sys-kernel/coreos-modules/files/commonconfig-4.14

```diff
@@ -514,6 +514,10 @@ CONFIG_SCSI_QLA_FC=m
 CONFIG_SCSI_LPFC=m
 CONFIG_SCSI_VIRTIO=m
 CONFIG_SCSI_DH=y
+CONFIG_SCSI_DH_RDAC=m
+CONFIG_SCSI_DH_HP_SW=m
```

Comment:
Per the [Kconfig text](https://github.com/torvalds/linux/blob/968edbd93c0cbb40ab48aca972392d377713a0c3/drivers/scsi/device_handler/Kconfig#L24-L26) (from 2008!) let's leave this one disabled.

Reply:
removed